wri_sites.install

```diff
@@ -186,9 +187,24 @@ function _install_default_program_hero() {
     'info' => 'Happening Now',
     'type' => 'hero',
     'langcode' => 'en',
-    'revision_id' => 100000,
+    'revision_id' => 100070,
     'reusable' => FALSE,
     'field_hero_type' => $paragraph,
   ]);
   $block->save();
 }
+
+/**
+ * Installs the Narrative Taxonomy block.
+ */
+function _install_narrative_taxonomy() {
```

Comment:
These are the piece that was missing for the program pages to load after a fresh install.